src/components/tile/tile.component.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from "prop-types";
 import propTypes from "@styled-system/prop-types";
 import { StyledTile, TileContent } from "./tile.style.js";
 
-const Tile = ({
+export const Tile = ({
```

Comment:
Do we need to use a named export for `Tile` from this file? I don't think this is used anywhere internally and the index file within `/src/components/tile` exports the component as a default export anyway.

Reply:
I think we won't see the prop tables properly in storybook without it